backend/satellite_tools/spacewalk-debug

```diff
@@ -130,6 +137,8 @@ elif [ -d /etc/apache2 ]; then
 fi
 cp -fapRd /etc/rhn $DIR/conf/rhn
 cp -fapRd /etc/sysconfig/rhn $DIR/conf/rhn/sysconfig
+# exclude schema upgrade files, as they are packaged
+rm -r $DIR/conf/rhn/sysconfig/rhn/schema-upgrade
```

Comment:
Hmm, I am not 100% sure if this is a good idea, because you never know if a customer messed around with it.
But I tend to agree that in such a case we might need to ask explicit for that content.

Reply:
This was initially a port from: https://bugzilla.suse.com/attachment.cgi?id=830902&action=diff#a/backend/satellite_tools/spacewalk-debug_sec2
I assume Moio opinion on removing this folder :)
Do you think this folder been removed should be explicitly communicated in the mailing list, so all developers be aware we don't will have it anymore and need to ask if needed?

Reply:
I echo @mcalmer's sentiment above - we only need to request this directory if the schema packages have been tampered with (and we have this information in the rpm verification files). So I would say this change is OK.